### PR TITLE
doc: Overall documentation improvments

### PR DESCRIPTION
**Pull request by jubianchi**

### CODE REVIEW

File: doc/installation.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ install this extension.
 To make things easy, we strongly encourage you to use [Homebrew](https://brew.sh/). It will let you install the most 
 recent PHP versions and development tools which you will need to compile the extension.
```
Comment (Hywan) [inline doc/installation.md]:
```diff
- PHP versions
+ PHP version
```

Comment (Hywan) [inline doc/installation.md]:
```diff
- tools which you
+ tools you
```

File: doc/installation.md [hunk 2]
```diff
@@ -114,7 +114,7 @@ Now open the file by double-clicking on it and your package manager should guide
 _Depending on your distro, you might get errors when installing the package (somrthing about unsupported OS). If you
```
Comment (Hywan) [inline doc/installation.md]:
```diff
- somrthing
+ something
```

File: doc/profiling.md [hunk 3]
```diff
@@ -1,4 +1,4 @@
-# prof-it - Profiling
+# Profiling
 
 In the following section, we will profile the following code:
```
Comment (Hywan) [inline doc/profiling.md]:
```diff
- we will profile the following code
+ we will profile this code
```

File: doc/profiling.md [hunk 4]
```diff
@@ -48,3 +48,17 @@ $profile->export(__DIR__);
 The `export` method will write the profile data to a file in the provided directory.
 
 ## Open your first profile
+
+Now open the client application and drop the file on the drop zone :
```
Comment (Hywan) [inline doc/profiling.md]:
```diff
- the drop zone :
+ the drop zone:
```

**Comment by jubianchi** [comment]
thanks @Hywan

**Comment by Hywan** [comment]
:+1:

**Pull request merged**